cnf-tests/pod-utils/cyclictest-runner/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"k8s.io/klog"
+	"os/exec"
+	"strconv"
+	"time"
+
+	podutils "github.com/openshift-kni/cnf-features-deploy/cnf-tests/pod-utils"
+)
+
+const cyclictestBinary = "/usr/bin/cyclictest"
+
+func main() {
+	klog.InitFlags(nil)
+
+	var rtPriority = flag.String("rt-priority", "1", "specify the SCHED_FIFO priority (1-99)")
+	var duration = flag.String("duration", "15s", "specify a length for the test run. Append 'm', 'h', or 'd' to specify minutes, hours or days.")
+	var threads = flag.String("threads", "", "number of threads: without NUM, threads = max_cpus. without -t default = 1")
+	var histogram = flag.String("histogram", "30", "dump a latency histogram to stdout after the run US is the max latency time to be be tracked in microseconds")
+	var interval = flag.String("interval", "1000", "  base interval of thread in us default=1000")
+	var cyclictestStartDelay = flag.Int("cyclictest-start-delay", 0, "Delay in second before running the cyclictest binary")
+
+	flag.Parse()
+
+	selfCPUs, err := podutils.GetSelfCPUs()
+	if err != nil {
+		klog.Fatalf("failed to get self allowed CPUs: %v", err)
+	}
+
+	if len(*threads) == 0 {
+		*threads = strconv.Itoa(selfCPUs.Size())
+	}
+
+	podutils.PrintNodeInformation()
+
+	if *cyclictestStartDelay > 0 {
+		time.Sleep(time.Duration(*cyclictestStartDelay) * time.Second)
+	}
+
+	cyclictestArgs := []string{
+		"-D", *duration,
```

Comment:
This will cause a segfault if one of these parameters is not set.
If we need all it's common practice to check against their presence and print out flag.Usage()

Reply:
Right, ignore what I just wrote